src/components/TopBar/TopBar.scss

```diff
@@ -0,0 +1,35 @@
+.top-bar {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 12.5rem;
+  gap: 1rem;
+
+  .heading {
+    font-size: 2rem;
+    font-weight: bold;
+  }
+
+  .date {
+    font-size: 0.875rem;
+    color: #666;
+  }
+
+  .workspace-selection {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    width: 100%;
+
+    .workspace-label {
+      font-size: 0.8rem;
+      color: #666;
+      font-weight: 300;
```

Comment:
More something we should look at in the future. I don't use Stylelint much, but thought styles would be ordered, unless the order somehow impacted the display. So I expected `font-size` and `font-weight` to be together, but having looked at this in VS Code and run stylelint manually against this branch, it doesn't. There is a [stylelint-order](https://www.npmjs.com/package/stylelint-order) that we could add, I'll look to do this in the starter project and get back to you on this. Not looking for anything to be done right now.